common/preformance.py

```
@@ -0,0 +1,1250 @@
+
+        This method wraps the standard Django TemplateView with cache_page,
+        using the configured cache timeout to cache the rendered template response.
+
+        Args:
+            **initkwargs: Initialization keyword arguments for the view.
+
+        Returns:
+            function: A Django view function with caching applied.
+        """
+        return cache_page(settings.CACHE_TTL)(
+            super(CachedTemplateView, cls).as_view(**initkwargs)
```

Ruff (0.12.2)

Use `super()` instead of `super(__class__, self)`

Remove `super()` parameters

(UP008)